pymmcore_plus/_tests/test_core.py

```diff
@@ -330,24 +330,30 @@ def test_get_objectives(core: CMMCorePlus):
 
 
 def test_guess_channel_group(core: CMMCorePlus):
-    chan_group = core.getChannelGroup()
-    assert core.getOrGuessChannelGroup() == chan_group
+
+    assert core.getOrGuessChannelGroup() == ["Channel"]
```

Comment:
Maybe keep the first check that it matches the core method? I think there's some value in that

Reply:
Sure, I've added it